Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -71,7 +71,7 @@ final class Navigator: CoreNavigator {
                                                alternativeRoutes: alternativeRoutes)
             }
             
-            self?.navigator.setRoutesFor(routesParams) { [weak self] result in
+            self?.navigator.setRoutesFor(routesParams, reason: .newRoute) { [weak self] result in
```

Comment:
Stub for future proper implementation in the scope of telemetry events integration.